runtime/drivers/admin/repo.go

```diff
@@ -72,7 +72,7 @@ func (h *Handle) ListRecursive(ctx context.Context, glob string, skipDirs bool)
 		}
 
 		// Track file (p is already relative to the FS root)
-		p = filepath.Join("/", p)
+		p = path.Join("/", p)
```

Comment:
Out of curiosity, why did we switched to `path` pkg from `filepath` pkg ?
Given these are operating system paths and not URL paths.

Reply:
I un-disabled a linter that checks if you use `/` with `filepath`, which is illegal because `filepath` requires you to use the OS path separator. In this case however, the `p` is obtained from the `doublestar` package, which always normalizes paths as Unix paths (even on Windows).